src/foremast/pipeline/create_pipeline_manual.py

```diff
@@ -72,7 +72,7 @@ def create_pipeline(self):
 
             # Create all pipelines
             for pipeline_dict in pipelines:
-                pipeline_dict.setdefault('application', self.app_name)
+                pipeline_dict.setdefault('application', self.app)
```

Comment:
This makes it a breaking change right?  so in a template `{{application}}` would need to become something like `{{application.app_name}}` instead?

Reply:
We do have that internally on a few manual templates e.g. Spinnaker pipeline json:

```
"application": "{{ application }}",
```